Flepper.Tests.Unit/QueryBuilder/Commands/WhereFilterTests.cs

```diff
@@ -242,60 +242,87 @@ public void ShoulContainWhereWithNotEqualToNull()
         [Fact]
         public void ShoulContainLikeContains()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").Contains("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("%abc%", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeStartsWith()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").StartsWith("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("%abc", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeEndsWith()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").EndsWith("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("abc%", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeContainsAndStartsWithAndEndsWith()
         {
-            var result = FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
-                .Where("field1").Contains("abc")
-                .And("field2").StartsWith("abc")
-                .And("field3").EndsWith("abc")
-                .Build()
+                .Where("field1").Contains("abc1")
+                .And("field2").StartsWith("abc2")
+                .And("field3").EndsWith("abc3")
+                .BuildWithParameters();               
+
+            result.Query.Should()
+                .Contain("@p0");
```

Comment:
You still must test the FULL SQL Query, do not use Contains

Reply:
Ok

Reply:
I did not understand

Reply:
We would like that you test the full sql query generated by fluent builder.

see example bellow
```
                  QueryResult result = FlepperQueryBuilder
                 .Select()
                 .From("table")
                 .Where("field1").Contains("abc1")
                 .And("field2").StartsWith("abc2")
                 .And("field3").EndsWith("abc3")
                 .BuildWithParameters(); 
                 
               result
               .Trim()
               .Should()
              .Be("SELECT * [table] WHERE field1 LIKE @p1 AND field2 LIKE @p2 AND field3 LIKE @p3")
```

Reply:
@gustavoferrazfontes Ok, Test done